.github/workflows/main.yml

```diff
@@ -91,7 +90,21 @@ jobs:
         repository: apache/incubator-nuttx-testing
         path: testing
 
+    - name: Docker Login
+      uses: azure/docker-login@v1
+      with:
+        login-server: docker.pkg.github.com
+        username: ${GITHUB_ACTOR}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Run Pull Container
```

Comment:
"Run builds already use ci-container at line 104, why we need another similar "Run Pull Container"?

Reply:
It is basically zero cost.  I left this so that we can see what the time for pull is, I'm worried that this is going to get quite large so I want to be able to keep an eye on it outside of test execution.  It will be local after this so the next step does not have to download again.